config/routes.rb

```diff
@@ -4,6 +4,16 @@
   devise_scope :user do
     get '/users/sign_out' => 'devise/sessions#destroy'
   end
+  
+  namespace :api, default: {format: :json} do 
+    namespace :v1,  default: {format: :json} do
+      resources :users, only: [:index, :show] do
+      resources :posts, only: [:index, :show, :create] do
+        resources :comments, only: [:index, :create]
+      end
```

Comment:
- [ ] <b>REQUIRED:</b> Like here. as I mentioned above, you forgot `likes`, and it is not even that directory. but it is outside.